'use strict';

/*
 * try/catch
 * Помилки фази інтерпретації обробити не можна, лише помилки фази виконання
 * Об'єкт помилки
 * try/catch працює ЛИШЕ з синхронним кодом
 * Обробка помилок в асинхронному коді
 */

// console.log('mdn')

// try {
// ,
// } catch(e) {
//     console.log(e)
// }

// console.log(1);

// setTimeout(() => {
//   try {
//     const number = 5;

//     number = 25;
//   } catch (err) {
//     console.dir(err);
//   }
// }, 1000);

// console.log(2);
